Build & publish docs CI is failing

Seems this has been failing for a while: https://github.com/fedimint/fedimint/actions/workflows/docs.yml

```
copying path '/nix/store/bxrxl669b0ylvd02bbsdymbh9i2fm0s5-source' from 'https://cache.nixos.org'...
copying path '/nix/store/fwpalmvh6cnphvvd6n362a5cq22ixniz-source' from 'https://cache.nixos.org'...
error: flake 'git+file:///home/runner/work/fedimint/fedimint?shallow=1' does not provide attribute 'packages.x[8](https://github.com/fedimint/fedimint/actions/runs/6773496178/job/18408486966#step:4:9)6_64-linux.debug.workspaceDocExport', 'legacyPackages.x86_64-linux.debug.workspaceDocExport' or 'debug.workspaceDocExport'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -19,7 +19,7 @@ jobs:
         continue-on-error: true
 
       - name: Build docs
-        run: nix build -L .#debug.workspaceDocExport
+        run: nix build -L .#ci.workspaceDocExport
 
       - if: github.repository == 'fedimint/fedimint'
         name: Deploy docs

